File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/7 - 9 and 10 March 2019/01_tennis_equipment.py
import math

price_tennis_rocket = float(input())
count_tennis_rockets = int(input())
count_kicks = int(input())

total_rockets = price_tennis_rocket * count_tennis_rockets
total_kicks = (price_tennis_rocket / 6) * count_kicks
total_extra = (total_rockets + total_kicks) * 0.2
total = total_rockets + total_kicks + total_extra

own = total / 8
sponsor = total * 7 / 8

print(f"Price to be paid by Djokovic {math.floor(own)}")
print(f"Price to be paid by sponsors {math.ceil(sponsor)}")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/03. Online Exam - 9 and 10 March 2019/01. Tennis Equipment.js
function tennisEquipment(params) {
    let priceRocket = Number(params.shift());
    let countRockets = Number(params.shift());
    let countShoes = Number(params.shift());

    let total_rockets = priceRocket * countRockets;
    let total_kicks = (priceRocket / 6) * countShoes;
    let total_extra = (total_rockets + total_kicks) * 0.2;
    let total = total_rockets + total_kicks + total_extra;

    let own = total / 8;
    let sponsor = total * 7 / 8;

    console.log(`Price to be paid by Djokovic ${Math.floor(own)}`)
    console.log(`Price to be paid by sponsors ${Math.ceil(sponsor)}`)
}

tennisEquipment(['850', '4', '2']);  // Expected output:
// Price to be paid by Djokovic 552
// Price to be paid by sponsors 3868

tennisEquipment(['1000', '3', '1']);  // Expected output:
// Price to be paid by Djokovic 475
// Price to be paid by sponsors 3325

tennisEquipment(['386', '7', '4']);  // Expected output:
// Price to be paid by Djokovic 443
// Price to be paid by sponsors 3108
